lib/config/pricing/src/index.js

```diff
@@ -148,7 +151,7 @@ const id = (oid, rtype, ppid, time, auth, cb) => {
   if(!rtype) {
     debug('Undefined resource type');
     return cb(undefined,
-      businessError(true, 'Pricing config - Undefined resource type'))
+      businessError('Pricing config - Undefined resource type'))
```

Comment:
It is true that the service name is irrelevant to the business person looking at the error because the endpoint to retrieve the error would have collector's service name on it. 

The reason why Undefined resource type is a bad error description is because there is a cascading effect. In the collector, the steps to enrich the usage doc is as follows:
1. get the resource type and get account information.
2. get metering plan id, rating plan id, and pricing plan id.
3. get the price.

Step 2 (get metering plan id, rating plan id, and pricing plan id) would automatically returns business error when step 1 (get resource type) fails . Step 3 (get price) would automatically returns business error when step 2 (pricing plan id) or step 1 (account information) fail.

Which made me question if I should stop the enriching process once an error is found in any of the steps because currently the errors are compiled at the end of steps 3. 

In this pull request, when there are errors when getting account information the collector would post:
```javascript
{
  start: 1420243200000,
  end: 1420245000000,
  organization_id: 'a3d7fe4d-3cb1-4cc3-a831-ffe98e20cf27',
  space_id: 'aaeae239-f3f8-483c-9dd0-de5d41c38b6a',
  consumer_id: 'external:bbeae239-f3f8-483c-9dd0-de6781c38bab',
  resource_id: 'test-resource',
  plan_id: 'basic',
  resource_instance_id: '0b39fa70-a65f-4183-bae8-385633ca5c87',
  measured_usage: [{
    measure: 'light_api_calls',
    quantity: 12
  }],
  error: true,
  reason: [
    'Unable to retrieve account information',
    'Undefined pricing country'
  ]
}
```
Undefined pricing country is the result of unable to retrieve account information.